gdal/apps/gdal_grid_lib.cpp

```diff
@@ -32,6 +32,7 @@
 #include "gdal_utils_priv.h"
 #include "commonutils.h"
 
+#include <cmath>
```

Comment:
Is anything from this header actually used?

Reply:
I added the header for the use of std::fabs.

Reply:
My bad, sorry. I missed the `fabs` indeed.